carbon-design-system/carbon-addons-iot-react
# Image card hotspots perimeter

Closes #2417

**Summary**
Image card hotspots where not respecting their placement when added on an outer edge. Namely on the bottom perimeter.

**Change List (commits, features, bugs, etc)**
Needed to keep track of the image size attributes so that the hotspots container can follow when the image changes dynamically with the zoom button or the displayOptions. (fit, fill, and stretch)

**Acceptance Test (how to verify the PR)**
 - Go to the dashboard editor and add an image (you should do both a portrait and a landscape image)
 - Open the hotspot editor and place hotspots around the edge
 - Save it and make sure they are where you expect them to be

You can use this image for a good landscape example
![image](https://user-images.githubusercontent.com/53092833/122610107-b51f1a00-d044-11eb-95e9-f5c1b9dbc487.png)


**Regression Test (how to make sure this PR doesn't break old functionality)**
Check the ImageCard story and make sure the static hotspots there are still present as expected.
